nilearn/image/tests/test_resampling.py

```diff
@@ -22,14 +22,27 @@
     resample_to_img,
 )
 from numpy.testing import (
+    assert_allclose,
     assert_almost_equal,
     assert_array_almost_equal,
     assert_array_equal,
+    assert_equal,
 )
 
+AFFINE_EYE = np.eye(4)
+
+ANGLES_TO_TEST = (0, np.pi, np.pi / 2.0, np.pi / 4.0, np.pi / 3.0)
+
+
+def _make_resampling_test_data():
+    rng = np.random.RandomState(42)
+    shape = (3, 2, 5, 2)
+    affine = AFFINE_EYE
+    data = rng.randint(0, 10, shape, dtype="int32")
```

Comment:
These `shape` and `data` values are repeated several times in this file. See if it makes sense to set to a constant variable or use this function within those tests

Reply:
I can try to use a fixture.

Reply:
done